src/ucp/rndv/proto_rndv.c

```diff
@@ -233,9 +228,6 @@ ucp_proto_rndv_ctrl_init(const ucp_proto_rndv_ctrl_init_params_t *params)
                                     UCP_DATATYPE_CONTIG, &mem_info, 1);
     }
 
-    /* Initialize estimated memory registration map */
-    ucp_proto_rndv_ctrl_get_md_map(params, &rpriv->md_map, &rpriv->sys_dev_map,
```

Comment:
why removed?
i think rndv/offload does not have to be derived from the common rndv protocol since in case of expected it will not require an operation on the remote side, and will be fully completed in HW.